### DISCUSSION TITLE
Change the remote URL?

### DISCUSSION THREAD

**Discussion by BatmanAoD** [jj-vcs/jj#3886] [Q&A]
Is there an equivalent to `git remote set-url`? I don't see it in `jj git remote help` or in the FAQ or git-compatibility doc.

**Comment by martinvonz** [comment]
No, I don't think we have that yet. I think it's just that no one has prioritized implementing it. Feel free to file a FR for tracking it.

**Reply by BatmanAoD** [reply]
Done: https://github.com/martinvonz/jj/issues/3901
